kotlin/internal/defs.bzl

```diff
@@ -36,6 +36,12 @@ KtJvmInfo = provider(
     },
 )
 
+KaptInfo = provider(
+    fields = {
+        "correctErrorTypes": "correct Error Types https://kotlinlang.org/docs/kapt.html#non-existent-type-correction",
```

Comment:
Can we follow snake case i.e `correct_error_types` here to stay consistent with rest of the args?